### ISSUE TITLE
OsmAnd live: rework rank computation

### ISSUE THREAD

**Issue by bagage** [osmandapp/osmandapp.github.io#135]
I'm not sure it's the proper place to address that issue, but I believe that the current rank formula for recipients reward (eg., the more changesets, the better rank) chould be improved:

When you look up first contributors on https://osmand.net/osm_live (sorted by OSM changes), the first ones are simply doing one changeset per node modified (adding tree one by one…).
In the mean time, someone that is importing 100K objects at once (buildings for instance) will do a single changeset. You don't want to have 100K changesets.

In my opinion, these extremes changeset (1K changesets of one node and 10K nodes modified at once) should be both valuated equally. But I believe it should also be equivalent to medium changesets (for instance 10 changesets of 10 modified objects). Maybe these "top contributors" will optimize their rank based on the new formula, but at least it would not encourage people to do lots of small changesets.

**Comment by vshcherb** [comment]
As of today we are counting only changesets. Also we are not trying to range users by top 10 or top 100, we are trying to find the 7 mostly ranked group. So if the user wants to be in top 1 group, he doesn't need to make 100K changesets he just need to have 192 (as of October).

Probably we will just our calculation based on number of edited objects in the new year.

**Comment by bagage** [comment]
> As of today we are counting only changesets.
I agree with everything you said. It was the point I was trying to issue. I do not know which type of users you want to reward the most, but I believe micro mapping tree one by one is not the best valuable thing OSM live buyers want.

> number of edited objects in the new year.
As I wanted to point out, I think it'll lead to many massive bad quality imports which is not desirable neither. I think best ranked contributors should be the ones who do a valuable job for buyers (eg. useful **and** properly done mapping).
